letsencrypt/tests/cli_test.py

```diff
@@ -10,8 +10,10 @@
 import mock
 
 from letsencrypt import account
+from letsencrypt import cli
```

Comment:
nit: In the rest of the repo, we only import the module we're testing in the specific functions where we need to do it.